Makefile

```diff
@@ -142,7 +142,7 @@ endif
 ##@ Development
 
 manifests: controller-gen kustomize authorino-manifests ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=authorino-operator-manager webhook paths="./..." output:crd:artifacts:config=config/crd/bases && $(KUSTOMIZE) build config/install > $(OPERATOR_MANIFESTS)
+	$(CONTROLLER_GEN) -h $(CRD_OPTIONS) rbac:roleName=authorino-operator-manager webhook paths="./..." output:crd:artifacts:config=config/crd/bases && $(KUSTOMIZE) build config/install > $(OPERATOR_MANIFESTS)
```

Comment:
This feels weird. It's printing controller-gen's help for everything now.

Is it really how we want to fix the SIGSEGV issue?

Reply:
I tried updating to new versions of controller gen and it fixes the panic issue but a new issue crops up that work with the -h. My thinking was go with -h for now to unblock quickly  and then try find a solution for other issue i think ill have to create a issue with them. My hunch is a versioning issue between go version and controller tools versions.  The panic is a known issue https://github.com/kubernetes-sigs/controller-tools/issues/888

Reply:
I've upgraded controller-gen to v0.15.0 and apparently it worked without hiccup.

Do you mind to give #188 a go please? Make sure to first `rm -f ./bin/*`. If it works for you, I'd rather merge that one first, then clean this one up a bit.

Reply:
Fixed in #188 and removed from this PR.

Reply:
Deadly happy days